apiserver/pkg/common/read_csv.go

```diff
@@ -20,37 +20,54 @@ import (
 	"io"
 )
 
-// ReadCSV function reads the data in lines from startLine and returns an error if there is an error.
-func ReadCSV(o io.Reader, startLine, lines int64) ([][]string, int64, error) {
+type ReadCSVResult struct {
+	Rows  [][]string `json:"rows"`
+	Total int64      `json:"total"`
+}
+
+// ReadCSV Reads the contents of a csv file by lines according to startLine and lines,
+// if cacheLines=0 then the total number of lines in the file will be counted,
+// otherwise the total number of lines in the file will not be counted and the speedup function returns.
+func ReadCSV(o io.Reader, startLine, lines, cacheLines int64) (ReadCSVResult, error) {
+	result := ReadCSVResult{}
```

Comment:
```golang
result := ReadCSVResult{
    Total:  cachedTotalLines
}
```

Reply:
https://github.com/kubeagi/arcadia/pull/385/files#diff-5d922defc7220baaf50dc44e878589147fea11600bfacbb79f30301654560991R50

`Total` is used to count the number of lines from the beginning, if the beginning is set to cachedTotallines, cachedTotalLines=0 is normal, but if cachedTotalLines is not 0, the total number of lines will be wrong